rfc/rfc-0005-customizable-consumption-offset.md

```diff
@@ -0,0 +1,67 @@
+# RFC-0005: Optional starting offset for processor inputs
+
+**Authors:** Eric Bottard
+
+**Status:**
+
+**Pull Request URL:** https://github.com/projectriff/riff/pull/1367
+
+**Superseded by:** N/A
+
+**Supersedes:** N/A
+
+**Related:** rfc-0003: developer utils, which also support viewing stream data from the earliest offset.
+
+
+## Problem
+Sometimes users want to replay data through functions, maybe because the function logic has changed, or maybe because downstream systems have lost state and that state needs to be re-created. The current Processor concept in riff doesn't support such a mechanism though, and always consumes data from the (current) end of streams.
+
+### Anti-Goals
```

Comment:
If this section is empty, we should probably leave out the heading. Otherwise, it might suggest that the following "Solution" is an anti-goal.